torch/_inductor/cpp_builder.py

```diff
@@ -322,10 +322,10 @@ def _remove_dir(path_dir: str) -> None:
         os.rmdir(path_dir)
 
 
-def _run_compile_cmd(cmd_line: str, cwd: str) -> bytes:
+def _run_compile_cmd(cmd_line: str, cwd: str) -> None:
     cmd = shlex.split(cmd_line)
     try:
-        status = subprocess.check_output(args=cmd, cwd=cwd, stderr=subprocess.STDOUT)
```

Comment:
I intend to keep current code. `check_output` need work with many OSs and many compilers. They will return different behaviors, even some error output messages. If we can't make sure we can make it better, we'd better keep current code.

Reply:
@xuhancn @Skylion007 

`status` is not a [status variable](https://docs.python.org/3/library/subprocess.html#subprocess.CompletedProcess.returncode): it is a byte string which merges `stdout` and `stderr`. At the very least, it should be correctly named.

But this result simply isn't used at all, anywhere in the code - indeed, it's not clear _how_ it could be used since we have no information about what the output might be, or even if that output is stable between runs of the compiler or different settings.

Keeping unused, trivial code around just in case it might be used in the future is not a good idea.

You will note that this variable has already consumed time from three different engineers, because its name gives the false impression that we had forgotten to check a status variable.

Reply:
I completely agree with @rec. It doesn't make any sense to use a misleading and useless variable here. We should remove it directly. @xuhancn, if you strongly believe that checking the compilation output is necessary for robustness, please submit another PR, and we will review it carefully.